android/src/main/java/com/brentvatne/react/ReactVideoView.java

```diff
@@ -410,6 +423,7 @@ public void setPausedModifier(final boolean paused) {
                 mProgressUpdateHandler.post(mProgressUpdateRunnable);
             }
         }
+        setProximitySensorEnabled(!paused);
```

Comment:
```suggestion
        setProximitySensorEnabled(!mpaused);
```
To keep with the style of the modifier